podman/domain/volumes.py

```diff
@@ -34,6 +34,33 @@ def remove(self, force: Optional[bool] = None) -> None:
             APIError: when service reports an error
         """
         self.manager.remove(self.name, force=force)
+    
+    def inspect(self) -> dict:
```

Comment:
I think in your squashing you added your changes twice, please just add the second part.

(make the change, `git commit -as --amend`, fix the message, then it should look better)

Reply:
Ehh... Something went clearly wrong here. Removed this section.